Codeinwp/themeisle-companion
# Remove Templates Directory

Removes the Template Directory. When the user wants to access the Template Directory, they get redirected to the Neve Starter Sites.

### Test instructions
- Activate the Template Directory Module from the Orbit Fox dashboard 
- Go to the Template Directory page from the Orbit Fox submenu
- Templates Directory should be available only when both Neve and Templates Patterns Collection plugin are activated.
- If one or both of them are not installed or activated, a notice should appear with buttons for installing and activating them
- After both are activated, the user should get redirected to the Neve Starter Sites page

Closes #715.
